ollama/deployment/src/main/java/io/quarkiverse/langchain4j/ollama/deployment/OllamaProcessor.java

```diff
@@ -1,7 +1,6 @@
 package io.quarkiverse.langchain4j.ollama.deployment;
 
-import static io.quarkiverse.langchain4j.deployment.LangChain4jDotNames.CHAT_MODEL;
-import static io.quarkiverse.langchain4j.deployment.LangChain4jDotNames.EMBEDDING_MODEL;
+import static io.quarkiverse.langchain4j.deployment.LangChain4jDotNames.*;
```

Comment:
We generally avoid star imports

Reply:
yes ou are right, I think it's my IDE.
I had to switch to Intellij because my VSCode was crashing when I try to use it to dev on the extension. So my settings are not quite perfect on my Intellij 😉

Reply:
➡️ https://github.com/quarkiverse/quarkus-langchain4j/pull/522/commits/efc0de2213cdc181d26d9377a46890c403b0642b